app/src/main/java/org/dhis2/usescases/reservedValue/ReservedValueViewHolder.java

```diff
@@ -8,16 +8,16 @@
 public class ReservedValueViewHolder extends RecyclerView.ViewHolder {
 
     private ItemReservedValueBinding binding;
-    private ReservedValueContracts.Presenter presenter;
+    private ReservedValuePresenter presenter;
 
-    public ReservedValueViewHolder(ItemReservedValueBinding binding, ReservedValueContracts.Presenter presenter) {
+    public ReservedValueViewHolder(ItemReservedValueBinding binding, ReservedValuePresenter presenter) {
         super(binding.getRoot());
         this.binding = binding;
         this.presenter = presenter;
 
     }
 
-    public void bind(ReservedValueContracts.Presenter presenter, ReservedValueModel dataElement){
+    public void bind(ReservedValuePresenter presenter, ReservedValueModel dataElement){
         //TODO cambiarlo en el xml tambien
```

Comment:
For the next time. If you see any comment, TODO or whatever in other language than english. Please change it :)

Reply:
thanks, I did not notice. Not sure if android studio change it automatically. I'll do it